app/src/main/java/de/westnordost/streetcomplete/quests/atm_deposit/AddATMDeposit.java

```diff
@@ -0,0 +1,45 @@
+package de.westnordost.streetcomplete.quests.atm_deposit;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import de.westnordost.streetcomplete.R;
+import de.westnordost.streetcomplete.data.osm.SimpleOverpassQuestType;
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder;
+import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataDao;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+import de.westnordost.streetcomplete.quests.YesNoQuestAnswerFragment;
+
+public class AddATMDeposit extends SimpleOverpassQuestType
+{
+	@Inject public AddATMDeposit(OverpassMapDataDao overpassServer) { super(overpassServer); }
+
+	@Override protected String getTagFilters()
+	{
+		return "nodes, ways with amenity=atm and !cash_in";
```

Comment:
I think there are no `way`-ATMs.

Reply:
There are 461 of them, and 392 of those also have a `building` tag. Some seem to be set up as kiosks, but I think others are tagged on the shop (or whatever) and ought to be their own nodes (and/or a re-tag to atm=yes).

Reply:
But according to the [wiki](https://wiki.openstreetmap.org/wiki/Tag:amenity=atm) this is not allowed...

Reply:
Now I look only for nodes